sparse/tests/test_core.py

```diff
@@ -153,6 +154,17 @@ def test_elemwise(func):
     assert_eq(func(x), func(s))
 
 
+@pytest.mark.parametrize('func', [operator.mul])
+def test_elemwise_binary(func):
+    x = random_x((2, 3, 4))
+    y = random_x((2, 3, 4))
```

Comment:
We might want to try a few shapes, in particular we should be sure to include a single-dimensional array.